common/kvstore/src/main/java/org/apache/spark/util/kvstore/LevelDBIterator.java

```diff
@@ -182,23 +193,36 @@ public boolean skip(long n) {
 
   @Override
   public synchronized void close() throws IOException {
-    db.notifyIteratorClosed(this);
+    db.notifyIteratorClosed(it);
     if (!closed) {
-      it.close();
-      closed = true;
-      next = null;
+      try {
+        it.close();
+      } catch (UncheckedIOException uncheckedIOException) {
```

Comment:
It should be impossible to catch an exception of type  `UncheckedIOException` here.`

Reply:
thanks，done